lib/cardsagainstdiscord/deck_football.go

```diff
@@ -15,7 +15,7 @@ func init() {
 			&PromptCard{Prompt: `I love %s so much I would lend him my children.`},
 			&PromptCard{Prompt: `Come place your bets, ladies and gentlemen! %s vs. %s. Who will win?`},
 			&PromptCard{Prompt: `What's worse then being a Spurs fan? %s.`},
-			&PromptCard{Prompt: `Rumors say if you say %s infront of a mirror 3x your club will win a quadruple.`},
+			&PromptCard{Prompt: `Rumors say if you say %s in fronr of a mirror 3x your club will win a quadruple.`},
```

Comment:
Neither the original nor the proposed text is correct.
```suggestion
			&PromptCard{Prompt: `Rumors say if you say %s in front of a mirror 3x your club will win a quadruple.`},
```